internal/controller/monitor/monitor.go

```diff
@@ -37,6 +38,10 @@ type JobHandler interface {
 func New(logger *zap.Logger, k8s kubernetes.Interface, cfg Config) (*Monitor, error) {
 	graphqlClient := api.NewClient(cfg.Token)
 
+	if cfg.PollInterval == 0 {
```

Comment:
1 second should be a lower bound, so let's do this:

```suggestion
	if cfg.PollInterval < time.Second {
```